metaflow/cli.py

```diff
@@ -802,7 +802,10 @@ def resume(
     write_run_id(run_id_file, runtime.run_id)
     runtime.print_workflow_info()
     runtime.persist_constants()
-    runtime.execute()
+    if clone_only:
```

Comment:
This seems to change the behavior only for our internal use case of clone first and then execute but it feels like this mechanism would be able to work even for a regular resume by first cloning and then continuing execution?

Reply:
talked offline, lets make this PR focusing on clone-only behavior first.